config/routes.rb

```diff
@@ -1,9 +1,8 @@
 Rails.application.routes.draw do
   resources :people
-  resources :events, only: [:show, :create, :new, :upcoming] do
-    get "/upcoming-events", to: 'events#upcoming', on: :collection
-  end
+  resources :events, only: [:show, :create, :new, :upcoming]
   resources :presentations, only: [:show, :create, :new]
 
   root 'welcome#index'
+  get "/upcoming-events", to: 'events#upcoming'
```

Comment:
To further add there are going to be multiple /events routes already so backtracking to /upcoming-events would be weird.